src/mjolnir/elevationbuilder.cc

```diff
@@ -210,8 +247,8 @@ void ElevationBuilder::Build(const boost::property_tree::ptree& pt) {
   // Spawn the threads
   for (auto& thread : threads) {
     results.emplace_back();
-    thread.reset(new std::thread(add_elevation, std::cref(pt), std::ref(tilequeue), std::ref(lock),
-                                 std::ref(sample), std::ref(results.back())));
+    thread.reset(new std::thread(add_elevations_to_multiple_tiles, std::cref(pt), std::ref(tilequeue),
```

Comment:
test coverage not required here as the whole method emerged from previously existing method, it was done just for reuse